CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
+- Changed display of graph notifications for settlement schoolSalary
```

Comment:
Hm. This PR deals with graduation rate and loan default rate, not the salary values. (I know because I'm working on them. :smile:)